### PR TITLE
Node path: Remove portfolio mention from ToDo

### LINKED ISSUE #27600: JS ToDo list: Remove portfolio from introduction
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

In the discord we advise (along with the associated command) to not think about portfolio worthy-ing projects, until much later than the organising code section where ToDo lives. However in the introduction to the page it mentions exactly that. 

I feel this is encouraging people to spend too long on areas that aren't the main focus of the task - since they'll be much better poised to make proper portfolio work later on, that will outshine ToDo

### Path

Node / JS

### Lesson Url

https://github.com/TheOdinProject/curriculum/blob/main/javascript/organizing_your_javascript_code/project_todo_list.md

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by damon314159**
## Because
It's a bit early in the curriculum to be putting effort into getting projects 'portfolio ready'

## This PR
Rewords the introduction

## Issue
Closes #27600

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/organizing_your_javascript_code/project_todo_list.md [hunk 1]
```diff
@@ -28,7 +29,7 @@ Before diving into the code, take a minute to think about how you are going to w
 8. We haven't learned any techniques for actually storing our data anywhere, so when the user refreshes the page, all of their todos will disappear! You should add some persistence to this todo app using the Web Storage API.
    1. `localStorage` ([docs here](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API)) allows you to save data on the user's computer. The downside here is that the data is ONLY accessible on the computer that it was created on. Even so, it's pretty handy! Set up a function that saves the projects (and todos) to localStorage every time a new project (or todo) is created, and another function that looks for that data in localStorage when your app is first loaded. Additionally, here are a couple of quick tips to help you not get tripped up:
```
Comment (MaoShizhong) [inline javascript/organizing_your_javascript_code/project_todo_list.md:30]:
```suggestion
   1. [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API) allows you to save data on the user's computer. The downside here is that the data is ONLY accessible on the computer that it was created on. Even so, it's pretty handy! Set up a function that saves the projects (and todos) to localStorage every time a new project (or todo) is created, and another function that looks for that data in localStorage when your app is first loaded. Additionally, here are a couple of quick tips to help you not get tripped up:
```
Would you mind fixing this link up as well please?
Note that the backticks should be dropped as including them makes the link visually hard to discern `from` normal inline code blocks [`like`](#) so (especially on the TOP website).

**Review by MaoShizhong — COMMENTED** [review]
Thanks for doing this. Would you also be able to fix up the linting issues in this file?
If you add a blank line after the div on line 13, you should find the linter suddenly realises there are plenty of list style errors (and is also why it didn't flag the `[docs here]` link error.

I've a PR open that'll add a lint rule to enforce this blank space to prevent these misses from occurring.

**Comment by damon314159** [comment]
@MaoShizhong all good now?

**Review by MaoShizhong — APPROVED** [review]
:rocket:

**Pull request merged**